pkg/plugin/v2/scaffolds/init.go

```diff
@@ -40,6 +40,9 @@ const (
 	ControllerToolsVersion = "v0.3.0"
 	// KustomizeVersion is the kubernetes-sigs/kustomize version to be used in the project
 	KustomizeVersion = "v3.5.4"
+	// ControllerRuntimeEnvTestVersion version to be used to download the envtest setup script
+	// todo: update the tag release when the next version of the project be released with this script
+	ControllerRuntimeEnvTestVersion = "master"
```

Comment:
we just spoke in the meeting. Shows that we have a release already for that. So, we need ping the tag. 
also, we cannot merge point for the master.

Reply:
I just checked that we have a new version to bump. https://github.com/kubernetes-sigs/controller-runtime/blob/v0.6.3/hack/setup-envtest.sh